app/src/main/java/net/aiscope/gdd_app/ui/mask/customview/MaskLayer.kt

```diff
@@ -182,14 +202,7 @@ class MaskLayer(private val imageMatrix: Matrix) {
     fun redoAvailable() = undoPendingPaths > 0
 
     fun getBitmap(): Bitmap {
-        return Bitmap.createBitmap(
-            currentStateBitmap.width,
-            currentStateBitmap.height,
-            Bitmap.Config.ARGB_8888
-        ).apply {
-            val canvas = Canvas(this)
-            drawPaths(canvas, removeAlpha = true)
-        }
+        return currentStateBitmap.apply { drawPaths(currentStateBitmapCanvas, removeAlpha = true) }
```

Comment:
@eduardb A change is needed to make sure we draw on the `initialBitmap` rather than a new empty one. Think it may be a mistake to use the the `currentStateBitmap` though, will have a look if I can stop using that.

Reply:
Alright updated implementation. Neater like this I think